packages/picker/README.md

```diff
@@ -393,6 +393,42 @@ When the `value` of an `<sp-picker>` matches the `value` attribute or the trimme
 </sp-picker>
 ```
 
+### Loading
```

Comment:
We should outline here that when `[loading]` a Pickers listbox cannot be opened, maybe:

> When given a `loading` attribute, an `<sp-picker>` will be delivered with an `<sp-progress-circle>` to visually outline that it is "loading" and it will not toggle open or display its `<sp-menu-item>` descendants until the attribute is removed.